src/Exception/Helper/UnsupportedConfigurationValueException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Nipwaayoni\Exception\Helper;
+
+class UnsupportedConfigurationValueException extends \Exception
```

Comment:
I think that `$message` argument could be removed from custom exceptions. What do you think?

Reply:
These exception class changes are actually on the other PR. I made this branch from that one while waiting on the review, but made the PR to master. I was fixing some other issues and didn't notice the unused `$message`. I'll clean all that up.

Reply:
I changed the base of this PR so the comments became outdated. More exception class changes are on the the other PR.